### ISSUE TITLE
Streamlining 

### ISSUE THREAD

**Issue by JohnRMoreau** [swcarpentry/sql-novice-survey#73]
tl;dr - Asking about two changes to the setup page before spending time on the pull request. 

Right now the current [student-facing setup lesson page](http://swcarpentry.github.io/sql-novice-survey/00-setup.html) does not include anything about how to quit SQLite. Novices encountering SQLite for the first time may become frustrated by not knowing how to quit the program back to the command line. Worse, they might assume the only way to quit is by exiting and restarting the shell. Is there a reason for this omission? If not, I'll put up a pull request to add the following snippet from the instructor page:

> To exit SQLite and return to the Bash shell, use .quit:
> ```
> sqlite> .quit
> $
> ```

Also, the setup page includes Python specific commands such as opening up an ipython notebook and running python commands. Do we require that participants\learners need Python installed and need to know Python concepts in order to do SQL? I understand that we normally teach Python and SQL in the same workshops. As we have more R workshops, we may need to teach SQL without teaching Python.  
__I propose that we should move the Python non sequitur to either a separate page in the SQL repo or to a page within the Python repo.__
Since there may be back history that I don't know, I wanted to ask for input before making the pull request.  Either way, I'm happy to prepare the markdown files.

**Comment by gvwilson** [comment]
`01-select.md` now has instructions on getting into *and out of* SQLite.
